pydicom/pixel_data_handlers/util.py

```diff
@@ -280,8 +296,12 @@ def apply_voi_lut(arr, ds, index=0):
         ``np.float64``. If neither are present then `arr` will be returned
         unchanged.
     index : int, optional
-        Where the VOI LUT Module contains multiple possible views, this is
+        When the VOI LUT Module contains multiple alternative views, this is
         the index of the view to return (default ``0``).
+    prefer_lut : bool
+        When the VOI LUT Module contains both *Window Width*/*Window Center*
+        and *VOI LUT Sequence*, if ``True`` (default) then apply the VOI LUT,
```

Comment:
-> then apply the VOI LUT Sequence
(they are both VOI LUTs)

Reply:
Are they? From wikipedia:

> In computer science, a lookup table is an array that replaces runtime computation with a simpler array indexing operation

This doesn't seem true for a windowing operation since we're applying some function to the already existing values. Or do you just mean they're both VOI LUTs in the sense they're both in the VOI LUT module?

Reply:
Hm, you are right. technically this is not a LUT, though you could implement it as one. The DICOM standard always speaks about the VOI LUT though (e.g. Modality LUT, VOI LUT and Presentation LUT), and I'm also used to this terminology, so I guess it is just a matter of what you are getting used to... anyway, this was a minor nitpick, it is clear enough as is either way.